cmd/server/server.go

```diff
@@ -42,19 +44,25 @@ var ServerCmd = &cobra.Command{
 		if err != nil {
 			log.Fatal("failed to connect to database:", err)
 		}
-		aRepo, err := repositoryMatch.NewRepository(db)
+		matchRepo, err := repositoryMatch.NewRepository(db)
 		if err != nil {
 			log.Fatal("failed to init repository:", err)
 		}
-		pRepo, err := repositoryPlayer.NewRepository(db)
+		playerRepo, err := repositoryPlayer.NewRepository(db)
 		if err != nil {
 			log.Fatal("failed to init repository:", err)
 		}
-		aMatch := appMatch.NewApp(aRepo)
-		aPlayer := appPlayer.NewApp(pRepo)
-		hMatch := handlerMatch.NewHandler(aMatch)
-		hPlayer := handlerPlayer.NewHandler(aPlayer)
-		s := httpServer.NewServer(hMatch, hPlayer)
+		teamRepo, err := repositoryTeam.NewRepository(db)
+		if err != nil {
+			log.Fatal("failed to init repository:", err)
```

Comment:
Este mensaje está bien pero yo pondría qué repositorio está fallando, algo como:

`failed to init team repository:` así si falla, te será más fácil saber cual. Lo mismo para los otros